pandas/tests/libs/test_lib.py

```diff
@@ -206,3 +206,10 @@ def test_no_default_pickle():
     # GH#40397
     obj = tm.round_trip_pickle(lib.no_default)
     assert obj is lib.no_default
+
+
+def test_is_list_like_iter_is_none():
```

Comment:
tests for this are in tests.dtypes.test_inference

Reply:
Aha, I only saw the doc tests and test_lib looked like the right place. I'll move it, thanks!